lsp/client/visualStudio/IdesLspPoc/ContentDefinitions/JsonContentType.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.Utilities;
+using System.ComponentModel.Composition;
+
+namespace IdesLspPoc.ContentDefinitions
+{
+    // This is used to associate our language server with JSON files
+    // JSON handling is sort of reserved by VS, this is the only way I found that activates our LSP and uses it with .json files
```

Comment:
Was MS forthcoming about this and were other content types listed or did you have to stumble upon this and search around a bit?  Curious how big an issue this might be for other content types.

Reply:
I had to stumble upon this through trial and error. I'm guessing each file type is implemented within Visual Studio as an internal extension, and there is likely some deviation for the behavior around each type.